src/sentry/static/sentry/app/components/group/suggestedOwnerHovercard.jsx

```diff
@@ -179,6 +215,22 @@ const OwnershipTag = styled(({tagType, ...props}) => <div {...props}>{tagType}</
   text-align: center;
 `;
 
+const ViewMoreButton = styled(({onClick, ...props}) => (
+  <div {...props}>
+    <ViewMoreLink onClick={onClick} />
+  </div>
```

Comment:
Should onClick be on the wrapper for a larger click area?

Reply:
Will do and make it a button